docs/keyboardShortcuts.md

```diff
@@ -17,11 +16,13 @@ These key sequences (one key followed by the other) are used to load (in the bro
 
 ## Create new ...
 
+On OS X, <kbd>⌘</kbd> replaces <kbd>Ctrl</kbd>.
+
 | **Item to create**| **Shortcut**|**Explanation**|
 | ------------- |:-------------:| --------- |
-| Issue|<kbd>Ctrl</kbd> + <kbd>I</kbd>|Loads the 'New Issue' page in the browser view|
-| Label|<kbd>Ctrl</kbd> + <kbd>L</kbd>|Loads the 'Labels' page in the browser view|
-| Milestone|<kbd>Ctrl</kbd> + <kbd>M</kbd>|Loads the 'New Milestone' page in the browser view|
+| Issue|<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>I</kbd>|Loads the 'New Issue' page in the browser view|
+| Label|<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>L</kbd>|Loads the 'Labels' page in the browser view|
+| Milestone|<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>M</kbd>|Loads the 'New Milestone' page in the browser view|
 | Panel (on right)|<kbd>Ctrl</kbd> + <kbd>P</kbd>| Create a new panel to the right of current panel|
 | Panel (on left)|<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>|Create a new panel to the left of current panel|
```

Comment:
Yes, the default should be on the right.
An HT user is expected to translate 'create new' to Ctrl+Shift (to be
consistent with other similar shortcuts) and 'panel' to P. then he will be
surprised to see the panel opening on left. That's the thing I'm worried
about. But it's ok, we can keep it as it is.